Ochowo/iReporter
# #162472441 endpoint validation

**What does this PR do?**
- Creates a validation test for API endpoints and enables delete request

**Description of the task  to be completed**
- Ensure the following endpoints are working as they should
    `GET/api/v1/redflags`
    `GET/api/v1/redflags/:id`
    `POST/api/v1/redflags`
    `PATCH/api/v1/redflags/:id/location`
    `PATCH/api/v1/redflags/:id/comment`
    `DELETE/api/v1/redflags/:id`

**How should this be tested manually**
After you clone the repo, cd into it and run `npm install` to install node modules. Make sure you have nodejs and express installed. Afterwards, run `npm start` to initiate the app.
- test all endpoints above using postman with `localhost:3000` as the host
- send empty inputs to test validation

**What are the relevant pivotal stories**
[#162472441](https://www.pivotaltracker.com/story/show/162472441)
